fh-mbaas-template-3node.json

```diff
@@ -1677,6 +1677,12 @@
       "value": "info"
     },
     {
+      "name": "MONGODB_PVC_SIZE",
+      "description": "The size of the volume for MongoDB Data",
+      "value": "50Gi",
+      "required": true
+    },
+    {
       "name": "SMTP_SERVER",
```

Comment:
Yeah, I'm not 100% sure about this one. I parameterized it because the parameter exists in the 1node-persistent template I think. It will default to 50Gi unless the default parameter value is explicitly overridden. I think it could be useful in cases where a user knows that they'll need a bigger database straight away for some reason. It could also be useful for us contributors, when trying the 3node template out, and we don't have a 50Gi PV.

I'm open to removing it though. @maleck13 do you have any strong thoughts on this?